requirements/base.in

```diff
@@ -0,0 +1,10 @@
+black==22.1.0
+build==0.7.0
```

Comment:
Missing `docker-compose`?

Reply:
`docker-compose` depends on `setuptools` -- which will generate some warnings in the requirements files. I personally use a system-wide docker-compose. Can we put a pin in this? (pun NOT intended)

Reply:
Note that if the warning is the same warning I'm thinking of around `--allow-unsafe`, the [`pip-compile` folks are considering deprecating it](https://github.com/jazzband/pip-tools/issues/989).